### PR TITLE
SB-23174: Adding script to update systemDefault default value for all framework and set compatibilityLevel=5 for all QuestionSet

### PR DESCRIPTION
**Pull request by amitpriyadarshi**

### CODE REVIEW

**Review by maheshkumargangula — DISMISSED** [review]

File: docs/cypher-scripts/release-3.8.0-all-objects.cypher [hunk 1]
```diff
@@ -114,4 +114,15 @@ match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['Collection', 'CollectionImage'
...
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['Collection', 'CollectionImage'] and exists(n.objectType) remove n.objectType;
+
+//Script to update framework with default systemDefault value as No
+
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE='Framework' and not exists (n.systemDefault) return count(n);
+
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE='Framework' and not exists (n.systemDefault) set n.systemDefault = 'No';
+
+// Scripts to set compatibilityLevel=5 for all QuestionSet
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['QuestionSet', 'QuestionSetImage'] and exists(n.compatibilityLevel) return n.compatibilityLevel, count(n.compatibilityLevel);
```
Comment (maheshkumargangula) [inline docs/cypher-scripts/release-3.8.0-all-objects.cypher:126]:
```suggestion
match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['QuestionSet', 'QuestionSetImage'] and exists(n.compatibilityLevel) return n.IL_FUNC_OBJECT_TYPE , n.compatibilityLevel, count(n.compatibilityLevel);
```

File: docs/cypher-scripts/release-3.8.0.cypher [hunk 2]
```diff
@@ -115,3 +115,14 @@ match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['Collection', 'CollectionImage'
...
 match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['Collection', 'CollectionImage'] and exists(n.objectType) remove n.objectType;
+
+//Script to update framework with default systemDefault value as No
+
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE='Framework' and not exists (n.systemDefault) return count(n);
+
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE='Framework' and not exists (n.systemDefault) set n.systemDefault = 'No';
+
+// Scripts to set compatibilityLevel=5 for all QuestionSet
+match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['QuestionSet', 'QuestionSetImage'] and exists(n.compatibilityLevel) return n.compatibilityLevel, count(n.compatibilityLevel);
```
Comment (maheshkumargangula) [inline docs/cypher-scripts/release-3.8.0.cypher:126]:
```suggestion
match (n:domain) where n.IL_FUNC_OBJECT_TYPE in ['QuestionSet', 'QuestionSetImage'] and exists(n.compatibilityLevel) return n.IL_FUNC_OBJECT_TYPE,  n.compatibilityLevel, count(n.compatibilityLevel);
```

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]

File: docs/cypher-scripts/release-3.8.0-all-objects.cypher (hunk 1 above)
Comment (amitpriyadarshi) [reply docs/cypher-scripts/release-3.8.0-all-objects.cypher]: update file with suggestion.

File: docs/cypher-scripts/release-3.8.0.cypher (hunk 2 above)
Comment (amitpriyadarshi) [reply docs/cypher-scripts/release-3.8.0.cypher]: update file with suggestion.

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**